app/assets/stylesheets/addons/_directional-values.scss

```diff
@@ -1,22 +1,29 @@
-// directional-property mixins are shorthands
-// for writing properties like the following
-//
-// @include margin(null 0 10px);
-// ------
-// margin-right: 0;
-// margin-bottom: 10px;
-// margin-left: 0;
-//
-// - or -
-//
-// @include border-style(dotted null);
-// ------
-// border-top-style: dotted;
-// border-bottom-style: dotted;
-//
-// ------
-//
-// Note: You can also use false instead of null
+/// directional-property mixins are shorthands
```

Comment:
This one needs `@charset "UTF-8";` at the top, right? And capitalizing "Directional-property"